stable/coredns/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: coredns
-version: 1.9.3
+version: 1.9.4
```

```suggestion
version: 1.10.0
```
this is like a new feature and not bug fixing please bump to 1.10.0